### PR TITLE
feat: cronjob implementation

### PR DESCRIPTION
**Pull request by boringContributor**

### CODE REVIEW

**Comment by boringContributor** [comment]
Current focus:

1. Target URL: As we do not have something like `enqueue` for a specific target, we do not provide the option `urlToOverrideWhenRunningLocalhost`. As the target is provided through the options, they are immediately updated when changed and therefore won't need this flag.

2. Creation of Cron: If use the Cron feature in a similar approach like the Queue, then it is a bit trickier to actually start the creation of the Cronjob, as the API handler is only executed when imported and used in another file. The current approach is to add `create`, `delete` and maybe `getCurrentCron` to the handler as we do with `enqueue` on the Queue and then trigger the execution there. The drawback is that the client has to manually trigger the Cron creation.

**Comment by boringContributor** [comment]
we should also keep this [issue](https://github.com/serverlessq-hq/serverlessq-nextjs/issues/9) in mind.

File: lib/cron/cron-handler-next.ts [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  ): Promise<void> {
+    return handler(req, res)
+  }
+
+  nextApiHandler.createOrUpdate = async (
+    options: Omit<CronOptions, 'nameOfCron' | 'target'>
+  ) => {
+    let target: string
+    if (!IS_VERCEL) {
+      console.log('Not running on Vercel. Probably on localhost')
```
Comment (AlessandroVol23) [inline lib/cron/cron-handler-next.ts:44]: Should we remove this log for prod?

**Comment by AlessandroVol23** [comment]
Cool! If we pass the wrong API Token we return 500, could we instead return 401. I think the API is returning 500 as well so we probably need to change it there as well.

**Pull request closed without merging**